core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/workspacemanager/HttpWorkspaceManagerDAO.scala

```diff
@@ -199,27 +199,68 @@ class HttpWorkspaceManagerDAO(apiClientProvider: WorkspaceManagerApiClientProvid
     )
   }
 
-  def createAzureStorageContainer(workspaceId: UUID, storageAccountId: Option[UUID], ctx: RawlsRequestContext) = {
+  override def createAzureStorageContainer(workspaceId: UUID,
+                                           storageContainerName: String,
+                                           storageAccountId: Option[UUID],
+                                           ctx: RawlsRequestContext
+  ) = {
     val storageContainerCreationParameters = storageAccountId match {
       case Some(_) =>
         new AzureStorageContainerCreationParameters()
-          .storageContainerName(s"sc-${workspaceId}")
+          .storageContainerName(storageContainerName)
           .storageAccountId(storageAccountId.get)
       case None =>
         new AzureStorageContainerCreationParameters()
-          .storageContainerName(s"sc-${workspaceId}")
+          .storageContainerName(storageContainerName)
     }
 
     getControlledAzureResourceApi(ctx).createAzureStorageContainer(
       new CreateControlledAzureStorageContainerRequestBody()
         .common(
-          createCommonFields(s"sc-${workspaceId}").cloningInstructions(CloningInstructionsEnum.DEFINITION)
+          createCommonFields(storageContainerName).cloningInstructions(CloningInstructionsEnum.DEFINITION)
         )
         .azureStorageContainer(storageContainerCreationParameters),
       workspaceId
     )
   }
 
+  override def cloneAzureStorageContainer(sourceWorkspaceId: UUID,
+                                          destinationWorkspaceId: UUID,
+                                          sourceContainerId: UUID,
+                                          destinationContainerName: String,
+                                          cloningInstructions: CloningInstructionsEnum,
+                                          ctx: RawlsRequestContext
+  ): CloneControlledAzureStorageContainerResult = {
+    val jobControlId = UUID.randomUUID().toString
+    getControlledAzureResourceApi(ctx).cloneAzureStorageContainer(
+      new CloneControlledAzureStorageContainerRequest()
+        .destinationWorkspaceId(destinationWorkspaceId)
+        .name(destinationContainerName)
+        .cloningInstructions(cloningInstructions)
+        .jobControl(new JobControl().id(jobControlId)),
+      sourceWorkspaceId,
+      sourceContainerId
+    )
+  }
+
+  override def getCloneAzureStorageContainerResult(workspaceId: UUID,
```

Comment:
For methods that are straight passthroughs (like this one), I haven't been creating unit test cases in HttpWorkspaceManagerDAOSpec. But perhaps that is just lazy?

Reply:
Sounds like being reasonable, to me: there isn't much to test, and any dependency updates that introduce breaking changes will either cause a compiler error, or likely not be caught by a unit test.